Source/OxyPlot/Axes/Rendering/AngleAxisRenderer.cs

```diff
@@ -51,13 +51,18 @@ public override void Render(Axis axis, int pass)
 
             var axisLength = Math.Abs(scaledEndAngle - scaledStartAngle);
             var eps = axis.MinorStep * 1e-3;
+
+            ScreenPoint someScreenPoint = ScreenPoint.Undefined;
+            double maxTextLength = double.MinValue;
+
             if (this.MinorPen != null)
             {
                 var tickCount = Math.Abs((int)(axisLength / axis.ActualMinorStep));
                 var screenPoints = this.MinorTickValues
                     .Take(tickCount + 1)
                     .Select(x => magnitudeAxis.Transform(magnitudeAxis.ClipMaximum, x, axis));
 
+                someScreenPoint = screenPoints.FirstOrDefault();
```

Comment:
Not tested, but note that the default with 0,0, not an undefined point, as it looks is expected: looks like this needs to be revised (or I could missing the point)